Ginger/Ginger/ALM/ALMItemsFieldsConfigurationPage.xaml.cs

```diff
@@ -171,7 +181,15 @@ private void fieldsWorker_Completed(object sender, RunWorkerCompletedEventArgs e
             LoadFieldsStatusLbl.Visibility = Visibility.Collapsed;
             grdQCFields.Visibility = Visibility.Visible;
             grdQCFields.DataSourceList = mItemsFields;
-            Reporter.ToUser(eUserMsgKey.StaticInfoMessage, "ALM Item Fields population is complete");
+            if (isExceptionToGetFields)
+            {
+                Reporter.ToUser(eUserMsgKey.StaticErrorMessage, "ALM item fields population failed");
```

Comment:
Update message to ask user to check error log in case of exception

Reply:
updated